src/invidious/views/template.ecr

```diff
@@ -150,6 +150,12 @@
                             <i class="icon ion-ios-wallet"></i>
                             <a href="https://invidious.io/donate/"><%= translate(locale, "footer_donate_page") %></a>
                         </span>
+                        <% if CONFIG.donate_url %>
+                        <span>
+                            <i class="icon ion-ios-wallet"></i>
+                            <a href="<%= CONFIG.donate_url %>"><%= translate(locale, "footer_instance_donate_page") %></a>
+                        </span>
+                        <% end %>
```

Comment:
Could you indent the `<span>` to match with the if block?

Reply:
isn't it already matching indent?
PS: why does prettier not support ecr files :(

Reply:
```suggestion
                        <% if CONFIG.donate_url %>
                              <span>
                                  <i class="icon ion-ios-wallet"></i>
                                  <a href="<%= CONFIG.donate_url %>"><%= translate(locale, "footer_instance_donate_page") %></a>
                              </span>
                        <% end %>
```